tests/starship/tests/setup_test.go

```diff
@@ -0,0 +1,51 @@
+package main
```

Comment:
Should this be run in CI? If so then you need to add it here down here https://github.com/cosmos/cosmos-sdk/blob/a6a283081bce9358d74e82f7080e62b97f98dc3f/.github/workflows/test.yml#L890.
However it won't run.